_sass/_objects--type.scss

```diff
@@ -0,0 +1,84 @@
+a {
+  @include simple-transition(color);
+}
+
+.p-content {
+  &, p, a {
+    font-size: 14px;
+    line-height: 1.7;
+
+    @include media-breakpoint-up(md) {
+      font-size: $font-size-base*1.25;
+    };
+  }
+}
+
+.p-lg {
+  font-size: $font-size-base;
+
+  @include media-breakpoint-up(md) {
+    font-size: $font-size-lg;
+  };
+}
+
+.h1-responsive {
+  font-size: $font-size-base*1.75;
+
+  @include media-breakpoint-up(md) {
+    font-size: $font-size-h1;
+  };
+}
+
+.h2-responsive {
+  font-size: $font-size-base*1.75;
+
+  @include media-breakpoint-up(md) {
+    font-size: $font-size-h2;
+  };
+}
+
+.illustrated-heading {
```

Comment:
It makes sense, could you put a comment above this class just to make clear where this is used?